src/chainstate/stacks/boot/pox.clar

```diff
@@ -362,33 +335,28 @@
 (define-read-only (can-stacks-stx (amount-ustx uint)
```

Comment:
While you're at it, can you change this to `can-stack-stx`?

Reply:
It looks like that could change in #1899 -- I'd rather not create a conflict here if so.